pkg/asset/internal/templates.go

```diff
@@ -157,22 +157,23 @@ spec:
         - apiserver
         - --bind-address=0.0.0.0
         - --secure-port=443
-        - --insecure-port=8080
+        - --insecure-port=0
         - --advertise-address=$(POD_IP)
         - --etcd-servers={{ range $i, $e := .EtcdServers }}{{ if $i }},{{end}}{{ $e }}{{end}}
         - --storage-backend=etcd3
         - --allow-privileged=true
         - --service-cluster-ip-range={{ .ServiceCIDR }}
-        - --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota
+        - --admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota
```

Comment:
We want this change -- but I don't think this is required for disabling the insecure port and we should move to a diff PR if so.